ledger/participant-state/src/main/scala/com/daml/ledger/participant/state/v2/SubmissionResult.scala

```diff
@@ -28,6 +28,7 @@ object SubmissionResult {
     override val description: String = s"Submission failed with error ${status.message}"
 
     def exception: StatusRuntimeException =
+      // TODO error codes: Adapt V2 ?
```

Comment:
Here we are already getting a Status, expecting that the `WriteService` is returning self-service error code formatted statuses. It should already contain the self-service error code. I'll look into it, thanks 👍